maquetado/scss/_tech.scss

```diff
@@ -0,0 +1,39 @@
+.tech {
+  background-color: $green;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  max-width: 100%;
+  padding: 14px;
+  position: relative;
+  text-align: center;
```

Comment:
Move this to `.tech-content`class style definition

Reply:
This attribute I didn't move because is necessary for the titles and content